### PR TITLE
Add export annotations as markdown button

### PR DESCRIPTION
**Pull request by eden-chan**
Example export:
```
## Chapter 1.1: Thinking Badly—Or Not Thinking at All?
- thinking for them. It’s as if we expect the inner voice in our head to say, “STOP! THIS IS A MOMENT WHEN YOU NEED TO THINK!”
			And because we don’t 
- we don’t realize that our brains have been hijacked by our biology, and that the outcome will go against what we seek. We don’t realize that hoarding information to gain an advantage is hurting the team. We don’t realize we’re conforming 
- n. Or you can cede control and execute a default behavior.
			The problem is, our default behavior often makes things worse.
			When someone slights us, we l (hello
)

## Chapter 1.2: The Emotion Default
- The Godfather is one of my favorite movies, (my annotation
)

## Chapter 1.3: The Ego Default
- the ego default prompts us to promote and protect our self-image at all costs. (very insightful!)
```
image

### CODE REVIEW

File: apps/reader/src/components/viewlets/AnnotationView.tsx [hunk 1]
```diff
@@ -51,11 +51,45 @@ const AnnotationPane: React.FC = () => {
...
+    a.download = 'exported_annotations.md'
+    a.click()
+  }
+
   return (
     <Pane headline={t('annotations')}>
       {keys(groupedAnnotation).map((k) => (
         <AnnotationBlock key={k} annotations={groupedAnnotation[k]!} />
       ))}
+      <div className="text-center mt-2.5">
```
Comment (pacexy) [inline apps/reader/src/components/viewlets/AnnotationView.tsx:90]: 1. it's better to use `Pane.actions` here instead, you can refer to `TocView`
2. hide if no items

**Review by pacexy — COMMENTED** [review]
I've fixed the version of `turbo`, so no changes need in `turbo.json`, just run `pnpm install` again.

**Comment by eden-chan** [comment]
image
Feel free to remove the download annotations as markdown blob

File: apps/reader/src/components/viewlets/AnnotationView.tsx [hunk 2]
```diff
@@ -51,8 +52,49 @@ const AnnotationPane: React.FC = () => {
...
+    }
+
+    // Copy to clipboard as markdown
+    const exportedAnnotationsMd = Object.entries(exported).map(([chapter, annotations]) => {
+      return `## ${chapter}\n${annotations.map((a) => `- ${a.text} ${a.notes ? `(${a.notes})` : ''}`).join('\n')}`
+    }).join('\n\n')
+    navigator.clipboard.writeText(exportedAnnotationsMd)
+
+    // Download as markdown
+    const blob = new Blob([exportedAnnotationsMd], { type: 'text/markdown' })
```
Comment (eden-chan) [inline apps/reader/src/components/viewlets/AnnotationView.tsx:79]: Feel free to remove this download as markdown download blob

File: apps/reader/src/components/viewlets/AnnotationView.tsx [hunk 3]
```diff
@@ -51,8 +52,49 @@ const AnnotationPane: React.FC = () => {
...
+    const url = URL.createObjectURL(blob)
+    const a = document.createElement('a')
+    a.href = url
+    a.download = 'exported_annotations.md'
+    a.click()
+  }
+
   return (
-    <Pane headline={t('annotations')}>
+    <Pane headline={t('annotations')} actions={Object.keys(groupedAnnotation).length > 0 ? [
```
Comment (eden-chan) [inline apps/reader/src/components/viewlets/AnnotationView.tsx:88]: conditionally render the copy annotation icon

Comment (eden-chan) [reply apps/reader/src/components/viewlets/AnnotationView.tsx:88]: image

File: apps/reader/src/components/viewlets/AnnotationView.tsx (hunk 2 above)
Comment (pacexy) [reply apps/reader/src/components/viewlets/AnnotationView.tsx:79]: To make it simple, can we just copy it as markdown?

Comment (eden-chan) [reply apps/reader/src/components/viewlets/AnnotationView.tsx]: Done! just removed the download as markdown snippet

Comment (eden-chan) [reply apps/reader/src/components/viewlets/AnnotationView.tsx]: @pacexy thank you for the feedback! excited to see this copy to clipboard as markdown on the site :)

**Review by pacexy — APPROVED** [review]

**Pull request merged**